### PR TITLE
🌱 modify scale e2e test defaults

### PR DESCRIPTION
**Pull request by ykakarap**
**What this PR does / why we need it**:

This PR modifies the scale E2E test defaults.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: test/e2e/data/infrastructure-inmemory/main/clusterclass-in-memory.yaml [hunk 1]
```diff
@@ -83,15 +83,49 @@ metadata:
   name: in-memory-control-plane
 spec:
   template:
-    spec: {}
+    spec:
```
Comment (sbueringer) [inline test/e2e/data/infrastructure-inmemory/main/clusterclass-in-memory.yaml:86]: Please also change the ClusterClass we publish

**Comment by sbueringer** [comment]
Thx!

/lgtm
/approve

**Review by killianmuldoon — COMMENTED** [review]
/area e2e-testing

**Pull request merged**